client/src/ui/CourseReviews.tsx

```diff
@@ -19,10 +19,11 @@ export default function CourseReviews({
     <div>
       {reviews.map((review) => (
         <ReviewCard
-          key={review._id}
-          info={review}
+          reviewId={review._id}
```

Comment:
I would be careful removing the `key` here because it's used in the `map()` to tell React what item to update (see here: https://beta.reactjs.org/learn/rendering-lists#why-does-react-need-keys)